store/sqlstore/user_store.go

```diff
@@ -1786,14 +1786,18 @@ func (us *SqlUserStore) GetUserLearningGroups(userId string, teamId string, lear
 				Channels.Id AS 'ChannelId',
 				Channels.Name AS 'ChannelName',
 				(
-				SELECT GROUP_CONCAT(Users.Username)
+				SELECT GROUP_CONCAT(
+									CONCAT(Users.id,',',Users.Username)
+									SEPARATOR ';'
```

Comment:
This query continues to make me sad, but now is still not the time to fix it.

Reply:
Yeah..I agree.